packages/itwin/reports-config-widget/src/widget/ReportsConfigUiProvider.tsx

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import type {
+  AbstractWidgetProps,
+  UiItemsProvider,
+} from "@itwin/appui-abstract";
+import {
+  AbstractZoneLocation,
+  StagePanelLocation,
+  StagePanelSection,
+  StageUsage,
+} from "@itwin/appui-abstract";
+import type { AccessToken } from "@itwin/core-bentley";
+import { IModelApp } from "@itwin/core-frontend";
+
+import * as React from "react";
+import ReportsContainer from "./components/ReportsContainer";
+
+export const REPORTS_CONFIG_BASE_URL = "https://api.bentley.com";
+
+export class ReportsConfigProvider implements UiItemsProvider {
+  public readonly id = "ReportsConfigProvider";
+  private readonly _accessToken?: AccessToken;
```

Comment:
wouldnt you want this to be a callback otherwise the token can get stale and expire

Reply:
Good point